### ISSUE TITLE
Embed syntastic directives in source

### ISSUE THREAD

**Issue by bedge** [vim-syntastic/syntastic#2054]
I have a list of dicts in python that describe a state machine config and have some peculiar formatting needs in order to remain readable as a sparsely populated event/state matrix.

Is it possible to disable specific checks for a specific block of code:
eg: E231 missing whitespace after ','

then re-enable all after the oddball format code block?

**Comment by lcd047** [comment]
TL;DR: no.

First, syntastic doesn't care about the contents of your files, only linters do.  So it's up to the particular linters you're using to do that or not.

Now, `E231 missing whitespace after ','` looks like a message from `pycodestyle` (formerly `pep8`).  According to the [manual][pep8], `pycodestyle` can disable messages by magic comments for some errors, but [not for `E321`][codes].

On syntastic's side there isn't much help either.  `quiet_messages` deals only with types of messages, not per-line, and `SyntasticCheckHook()` can't remove messages.

You could comment out that section just for linting, but that becomes awkward when you want to run the code.  Or you could just disable that message, and re-enable it only at the end, when everything else is done.  It's just a style tip anyway.

[pep8]: http://pycodestyle.pycqa.org/en/latest/
[codes]: http://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

**Comment by bedge** [comment]
Appreciate the detail. Thank you.
